dnf5/commands/download/download.cpp

```diff
@@ -122,22 +122,15 @@ void DownloadCommand::run() {
                 throw GoalResolveError(transaction);
             }
         }
-        download_pkgs.reserve(transaction.get_transaction_packages().size());
-        for (auto & tspkg : transaction.get_transaction_packages()) {
-            if (transaction_item_action_is_inbound(tspkg.get_action()) &&
-                tspkg.get_package().get_repo()->get_type() != libdnf::repo::Repo::Type::COMMANDLINE) {
-                download_pkgs.push_back(tspkg.get_package());
-            }
-        }
+        transaction.download();
```

Comment:
This modification makes failing test for download command. The problem is that the original implementation  downloaded packages into current directory, but the new implementation downloads them to cache of repository. There are two possibilities - to keep the original code or to change `destdir` for each repository to current directory ".".

Reply:
Oh I didn't see that, thanks. I think it will be best for now to keep the existing implementation. Earlier today @jan-kolarik suggested changing the API to add a setter for specifying a transaction's download destination, rather than having the destination dir as an argument to `transaction.download()`. So I think he's planning on changing things soon, might as well keep it as-is for now.